server/pulp/server/async/scheduler.py

```diff
@@ -6,18 +6,20 @@
 import platform
 import threading
 import time
+import uuid
+import mongoengine
```

Comment:
for pep8, this should be in the second group of imports. The first group is reserved for standard library imports.